### PR TITLE
fix(components): [input] error type(with typescript)

### PR DESCRIPTION
**Pull request by lequangdongg**
Please make sure these boxes are checked before submitting your PR, thank you!
Screen Shot 2023-03-30 at 10 03 22 PM

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/input/src/input.vue [hunk 1]
```diff
@@ -80,7 +80,7 @@
             </el-icon>
             <span v-if="isWordLimitVisible" :class="nsInput.e('count')">
               <span :class="nsInput.e('count-inner')">
-                {{ textLength }} / {{ attrs.maxlength }}
+                {{ textLength }} / {{ props.maxlength }}
```
Comment (tolking) [inline packages/components/input/src/input.vue:83]:
```suggestion
                {{ textLength }} / {{maxlength }}
```

File: packages/components/input/src/input.vue [hunk 2]
```diff
@@ -132,7 +132,7 @@
         :style="countStyle"
         :class="nsInput.e('count')"
       >
-        {{ textLength }} / {{ attrs.maxlength }}
+        {{ textLength }} / {{ props.maxlength }}
```
Comment (tolking) [inline packages/components/input/src/input.vue:135]:
```suggestion
        {{ textLength }} / {{maxlength }}
```

**Comment by lequangdongg** [comment]
Hi @tolking, I have updated your suggestion

**Review by tolking — APPROVED** [review]

File: packages/components/input/src/input.ts [hunk 3]
```diff
@@ -159,6 +159,12 @@ export const inputProps = buildProps({
     type: Boolean,
     default: true,
   },
+  /**
+   * @description maxlength
```
Comment (chenxch) [inline packages/components/input/src/input.ts:163]: It is recommended to keep the description consistent with the documentation.
`the max length`

**Comment by tolking** [comment]
Thanks for your contribution. Sorry, this issue seems to be fixed by another PR

**Pull request closed without merging**